lib/Runtime/Library/AtomicsObject.cpp

```diff
@@ -0,0 +1,270 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#include "RuntimeLibraryPch.h"
+
+#define ATOMICS_FUNCTION_BEGIN_WRAPPER(length, methodName) \
```

Comment:
agreed.

---
In reply to: [77402371](https://github.com/Microsoft/ChakraCore/pull/1533#discussion_r77402371) [](ancestors = 77402371)